plugins/check_curl.c

```diff
@@ -297,6 +297,7 @@ main (int argc, char **argv)
 
 int verify_callback(int preverify_ok, X509_STORE_CTX *x509_ctx)
 {
+	(void) preverify_ok;
```

Comment:
Could also be marked unused with something like:

_GL_ATTRIBUTE_UNUSED __attribute__ ((__unused__))

(but that one is from gnulib, so not sure, we should use that one).

The (void)PARAM approach works too.